docs/walk-through/loops.md

```diff
@@ -2,12 +2,13 @@
 
 When writing workflows, it is often very useful to be able to iterate over a set of inputs, as this is how argo-workflows can perform loops.
 
-There are two basic ways of running a template multiple times.
+There are three basic ways of running a template multiple times.
 
 - `withItems` takes a list of things to work on. Either
     - plain, single values, which are then usable in your template as '{{item}}'
     - a JSON object where each element in the object can be addressed by it's key as '{{item.key}}'
 - `withParam` takes a JSON array of items, and iterates over it - again the items can be objects like with `withItems`. This is very powerful, as you can generate the JSON in another step in your workflow, so creating a dynamic workflow.
+- `withSequence` take a Sequence, and convert it into a numeric sequence.
```

Comment:
this is arguably simpler than `withItems`, so maybe we should move this to be the first of the three in this list and in the examples below.

Reply:
also the definition you gave `withSequence` is currently recursive "Sequence [...] convert [...] sequence". it also has grammar and casing issues.

```suggestion
- `withSequence` iterates over a sequence of numbers.
```

- also simpler and more direct, per [k8s style guide](https://kubernetes.io/docs/contribute/style/style-guide/#use-simple-and-direct-language)